nasa/openmct
# Reimplement hyperlink vue

Fixes issue https://github.com/nasa/openmct/issues/4056

### All Submissions:


### Author Checklist


### Testing instructions
1. On openmct, create a hyperlink by clicking on the '+Create' dropdown menu on top left.
2. On the create hyperlink window, add URL and text to display. Pick either link or button for display format and open in new tab or this tab.
3. Verify that the hyperlink has the expected style (either link or button) and it opens the tab in the expected way.
